tsconfig.json

```diff
@@ -1,22 +1,23 @@
 {
 	"compilerOptions": {
 		"module": "commonjs",
-		"target": "es6",
+		"target": "es2019",
 		"outDir": "out",
 		"alwaysStrict":true,
 		"lib": [
-			"es6"
+			"es2019",
 		],
 		"sourceMap": true,
 		"rootDir": ".",
-		"strict": true   /* enable all strict type-checking options */
+        "useUnknownInCatchVariables": false,
+		"strict": true,   /* enable all strict type-checking options */
 		/* Additional Checks */
 		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
 		// "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
-		// "noUnusedParameters": true,  /* Report errors on unused parameters. */
+		// "noUnusedParameters": true,  /* Report errors on unused parameters. */,
 	},
 	"exclude": [
-		"node_modules",
+		"node_modules/**",
```

Comment:
Looks good, but also for my learning experience what inspired this. Squiggle recommended or what happened. CMake Tools may need the same.

Reply:
Good question, I did this to ensure that building failures didn't happen from node_modules, but I realized that the typescript dependency needed to be updated, this change can likely be reverted.

Reply:
Just tested and this isn't needed, reverting.